### PR TITLE
Feature/1 update coding standard

### PR DESCRIPTION
**Pull request by LarsSkovslund**
This PR aligns default repo structure and coding standards.

### CODE REVIEW

**Review by rickykaare — APPROVED** [review]

File: .github/workflows/post-integration.yml [hunk 1]
```diff
@@ -40,11 +40,14 @@ jobs:
         with:
           java-version: 1.11
 
-      - name: 📋 Prepare build for unit tests
-        run: |
-          dotnet clean -c Debug
-          dotnet restore
-          dotnet build -c Debug /p:UseSourceLink=true --no-restore
+      - name: 🧹 Clean
+        run: dotnet clean -c Release && dotnet nuget locals all --clear
```
Comment (egil) [inline .github/workflows/post-integration.yml:44]: Why is this needed. Each run of a workflow gets a clean file system to work with, as far as I know.

File: .github/workflows/post-integration.yml [hunk 2]
```diff
@@ -40,11 +40,14 @@ jobs:
...
+
+      - name: 🔁 Restore packages
+        run: dotnet restore
+
+      - name: 🛠️ Build
+        run: dotnet build -c Release --no-restore /p:UseSourceLink=true
```
Comment (egil) [inline .github/workflows/post-integration.yml:50]:
```suggestion
        run: dotnet build -c Release --no-restore /p:UseSourceLink=true /p:ContinuousIntegrationBuild=true
```

Pretty sure it is best practise to set the `p:ContinuousIntegrationBuild=true` during build or `dotnet pack`.

File: .github/workflows/post-integration.yml (hunk 1 above)
Comment (perkops) [reply .github/workflows/post-integration.yml:44]: Apparently it does not. We had alot of issues with this.

File: .github/workflows/post-integration.yml (hunk 2 above)
Comment (LarsSkovslund) [reply .github/workflows/post-integration.yml:50]: This is part of the `Directory.Build.props` file under `src`.
